cmd/create/cluster/cmd.go

```diff
@@ -1493,6 +1501,37 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
+	fips := args.fips
+	if interactive.Enabled() && fips {
+		fips, err = interactive.GetBool(interactive.Input{
+			Question: "Enable FIPS support",
+			Help:     cmd.Flags().Lookup("fips").Usage,
+			Default:  fips,
+		})
+		if err != nil {
+			reporter.Errorf("Expected a valid fips value: %v", err)
+			os.Exit(1)
+		}
+	}
+
+	etcdEncryption := args.etcdEncryption
+	if interactive.Enabled() && !fips {
```

Comment:
why only !fips ? if that is the case  will it help to add the validation 
```
if fips && cmd.Flags().Changed("etcd-encryption") && !etcdEncryption {
		reporter.Errorf("etcd encryption cannot be disabled on clusters with FIPS mode")
		os.Exit(1)
	}
```

before as a warning and give them interactive mode ?  or is it too much

Reply:
No, the interactive mode for `etcd-encryption` is hidden if they set `--fips`, however, if the user sets something like `--fips --etcd-encryption=false` that's when this error should show up